Nos/Views/NoteOptionsButton.swift

```diff
@@ -40,20 +40,22 @@ struct NoteOptionsButton: View {
                     analytics.copiedNoteIdentifier()
                     copyMessageIdentifier()
                 }
-                Button(Localized.copyNoteText.string) {
-                    analytics.copiedNoteText()
-                    copyMessage()
-                }
                 Button(Localized.copyLink.string) {
                     analytics.copiedNoteLink()
                     copyLink()
                 }
-                Button(Localized.viewSource.string) {
-                    analytics.viewedNoteSource()
-                    showingSource = true
-                }
-                Button(Localized.reportNote.string, role: .destructive) {
-                    showingReportMenu = true
+                if !note.isStub {
```

Comment:
This is a little feature that allows us to copy the ID of notes even if they are only stubs, which is helpful for debugging.